File: pgm/ABC_Java/ABC_220_A.java
import java.io.*;
// import java.math.*;
// import java.util.*;
// import java.util.Map.*;
// import java.util.stream.*;

public class Main {
    public static void main (String[] args) throws Exception {
        BufferedReader reader =
            new BufferedReader (new InputStreamReader (System.in));
        final String[] arr = reader.readLine().split (" ");
        final int no = -1;
        final int zero = 0;
        final int A = Integer.parseInt (arr[0]);
        final int B = Integer.parseInt (arr[1]);
        final int C = Integer.parseInt (arr[2]);
        final int multiple = A + C - A % C;
        if (A % C == zero) {
            System.out.println (A);
        } else if (A <= multiple && multiple <= B) {
            System.out.println (multiple);
        } else {
            System.out.println (no);
        }
        return;
    }
}

File: pgm/ABC_Kotlin/ABC_220_A.kt
// import java.io.BufferedOutputStream
// import java.io.BufferedReader
// import java.io.DataInputStream
// import java.io.FileInputStream
// import java.io.IOException
// import java.io.InputStreamReader
// import java.io.PrintWriter
// import java.lang.StringBuilder
// import java.time.LocalTime
// import java.time.format.DateTimeFormatter
// import java.util.Scanner
// import java.util.StringTokenizer

// import kotlin.math.*

const val space: String = " "
const val one: Int = 1
const val zero: Int = 0

fun main() {
    val (A: Int, B: Int, C: Int) = readln().split(space).map { it.toInt() }
    val multiple: Int = A + C - A % C
    if (A % C == zero) {
        println(A)
    } else if (multiple <= B) {
        println(multiple)
    } else {
        println(-one)
    }
    kotlin.system.exitProcess(0)
}